shpool/tests/data/motd_env_test_script.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "TERM=${TERM}"
+
+sleep 5
```

Comment:
this is going to increase test time. is 5 seconds needed?

is there any chance this test might be repeated over multiple parameters in the future?

I think this is fine, just want to make sure we don't have any issues that may reduce feasibility at any increase in scale